mfx/beamline.py

```diff
@@ -82,9 +82,9 @@
     from mfx.om import *
     om = OM()
 
-with safe_load('align'):
-    from mfx.optimize.align import *
-    align = Align()
+with safe_load('beam'):
+    from mfx.optimize.beam import *
```

Comment:
Question/minor suggestion: maybe unrelated to the goals of this PR, is there a reason this a `*` import?
```suggestion
    from mfx.optimize.beam import Beam
```

Reply:
thanks Zach, I like the idea of being a bit more specific about imports.

Reply:
Not sure. I am actually not sure what this file is doing, and was going to check in with you regarding this. Is it just loading things into the python environment at activation?